RULES.md

```diff
@@ -274,3 +274,14 @@ Field `parent_station` must be empty when `location_type` is 2.
 
 #### References:
 * [stops.txt specification](http://gtfs.org/reference/static/#stopstxt)
+
+### DuplicateRouteNameNotice
+
+All routes should have different `routes.route_long_name`. If routes have the same `routes.route_long_name`, they must be different routes serving different areas; and must not be different trips of the same route or different directions of the same route.
```

Comment:
I'd suggest being clearer here for the behavior of the validator vs. a caveat for when the notice may be detected in valid cases.

For example:

>All routes should have different `routes.route_long_name` - if two `routes.route_long_name` are the same, and the two routes belong to the same agency, a notice is generated.
>
>Note that there may be valid cases where routes may have the same `routes.route_long_name` and this notice can be ignored. For example, routes may have the same `routes.route_long_name` if they serve difference areas. However,  they must not be different trips of the same route or different directions of the same route - these cases should always have unique `routes.route_long_name`.

Reply:
> Note that there may be valid cases where routes may have the same routes.route_long_name and this notice can be ignored.

The code is actually written in a way that such a situation does not generate notice. Does  that mean that we should generate the notice anyway? @barbeau

Reply:
@lionel-nj Which case does not generate a notice? We aren't checking if routes serve different areas...

Reply:
My bad -  I got confused, please disregard my previous comment @barbeau.